docker-compose.prod.yml

```diff
@@ -19,6 +19,10 @@ version: '3.7'
 services:
   caddy:
     image: caddy:2
+    env_file:
+      # This should contain a list of env vars for configuring Firezone.
+      # See https://docs.firezone.dev/reference/env-vars for more info.
+      - ${FZ_INSTALL_DIR:-.}/.env
```

Comment:
Hm wait do we want to load the Firezone env vars into the `caddy` service? I fear it may cause an env var collision or increase attack surface unnecessarily.

Reply:
It should not make a collision, but we need to know phoenix port in the Caddy service. 

If you manage to get into Caddy that means you can listen to all the traffic and do pretty much whatever you want (eg. intercept admin cookies and do any sort of requests), so if we want to decrease attack surface - it should be simply removed.

Reply:
I believe docker-compose will automatically load env vars to use in the docker-compose.yml file from a `.env` in the current directory, so the service shouldn't actually need to use `PHOENIX_PORT` directly -- it's populated into the Caddyfile inline here.

Reply:
You are right, that's how TLS_OPTS worked